content/4_Gold/LIS.mdx

```diff
@@ -192,6 +192,72 @@ def find_lis(arr: List[int]) -> int:
 
 </LanguageSection>
 
+### Fast Solution (RMQ Data Structures)
+
+(By Aryansh Shrivastava) 
+
+**Time Complexity**: $\mathcal O(N \log N)$, but perhaps a constant factor slower than binary search. 
```

Comment:
I don't think you need to mention the constant factor, since a BIT should run at about the same speed as a normal binary search (if not faster)